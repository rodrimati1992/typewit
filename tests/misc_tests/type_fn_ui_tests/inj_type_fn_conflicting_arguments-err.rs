typewit::inj_type_fn!{
    struct Foo;

    impl u8  => u16;
    impl<T> T => Vec<T>;
}


fn main(){}