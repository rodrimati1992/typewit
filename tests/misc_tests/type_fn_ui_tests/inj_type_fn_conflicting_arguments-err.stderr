error[E0119]: conflicting implementations of trait `TypeFn<u8>` for type `Foo`
 --> tests/misc_tests/type_fn_ui_tests/inj_type_fn_conflicting_arguments-err.rs:1:1
  |
1 | / typewit::inj_type_fn!{
2 | |     struct Foo;
3 | |
4 | |     impl u8  => u16;
5 | |     impl<T> T => Vec<T>;
6 | | }
  | | ^
  | | |
  | |_first implementation here
  |   conflicting implementation for `Foo`
  |
  = note: this error originates in the macro `$crate::__impl_with_span` which comes from the expansion of the macro `typewit::inj_type_fn` (in Nightly builds, run with -Z macro-backtrace for more info)
